jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/AbstractSqlDialect.java

```diff
@@ -14,11 +15,31 @@
  * Abstract implementation of a dialect. We recommend that every dialect should extend this abstract class.
  */
 public abstract class AbstractSqlDialect implements SqlDialect {
+    // One of the following needs to be set
+    static final String SCHEMA_NAME_PROPERTY = "SCHEMA_NAME";
+    public static final String CONNECTION_NAME_PROPERTY = "CONNECTION_NAME";
+    static final String CONNECTION_STRING_PROPERTY = "CONNECTION_STRING";
+    static final String USERNAME_PROPERTY = "USERNAME";
+    static final String PASSWORD_PROPERTY = "PASSWORD";
```

Comment:
Take the property access from `AdapterProperties`. This way you don't need those constants.

Reply:
fixed